x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_panel.tsx

```diff
@@ -43,13 +44,13 @@ const ShortenedHorizontalRule = styled(EuiHorizontalRule)`
 const shouldShowStreamsByDefault = (
```

Comment:
it would be great to test this logic, maybe you could move this to a helper function so it can be tested without the component